exp/runtime/hooks/api/v1alpha1/common_types.go

```diff
@@ -20,6 +20,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// PendingHooksAnnotation is the annotation used to keep a track of pending runtime hooks.
+const PendingHooksAnnotation = "hooks.x-cluster.k8s.io/pending-hooks"
```

Comment:
I'm +1 for moving it to runtimev1, because that's the object the annitation it is getting set at.

Reply:
> that's the object the annitation it is getting set at.

The annotation is set at clusterv1.Cluster. But Runtime SDK isn't hard-coded to cluster so conceptually it could also be added to other objects in the feature, e.g. like Machine, InfraMachine, ControlPlane, ...